protocol/messenger_filter_init.go

```diff
@@ -24,7 +24,16 @@ func (m *Messenger) InitFilters() error {
 	rand.Seed(time.Now().Unix())
 
 	// Community requests will arrive in this pubsub topic
-	if err := m.SubscribeToPubsubTopic(wakuv2.DefaultNonProtectedPubsubTopic(), nil); err != nil {
+	// TODO depracate
```

Comment:
I noticed that there are a few todos in the PR, would be good to have issue linked here to resolve the todos.